requirements/test.txt

```diff
@@ -3,4 +3,3 @@
 pytest
 pytest-cov
 pytest-pep8
-tables
```

Comment:
Why this change?

Reply:
It seems to be failing when we add tables. I'll make the changes again to show them!